.github/actions/rpm-delivery/action.yml

```diff
@@ -78,15 +92,19 @@ runs:
         fi
 
         # Deliver based on inputs
-        for ROOT_REPO_PATH in "rpm-standard" "rpm-standard-internal"; do
+        #for ROOT_REPO_PATH in "rpm-standard" "rpm-standard-internal"; do
```

Comment:
```suggestion
```

Reply:
I m updating this , since the line 96 needs to be removed as well